lagunita/cms/static/sass/studio-main-v1-rtl.scss

```diff
@@ -0,0 +1,3 @@
+@import 'cms/static/sass/studio-main-v1-rtl';
+
+@import 'partials/overrides';
```

Comment:
This all seems like a lot of machinery to pull in a width override..

Given that `su-identity.css`:
-  is already a file we own/control
- `blame`s all its commits to you and Nate
  - including some from w/in the past year

it seems like we should just add our override to the existing file..